clients/web/package.json

```diff
@@ -9,6 +9,7 @@
     "@emotion/css": "11.13.5",
     "@emotion/react": "11.14.0",
     "@freestar/pubfig-adslot-react-component": "3.6.2",
+    "@mozilla-services/ads-sdk": "git+https://git@github.com:mozilla-services/ads-sdk.git",
```

Comment:
This doesn't seem to be working for me, it's giving permissions errors. I think it doesn't know to use my ssh keys for this remote like it does for ads-sdk? 

I did try cloning ads-sdk separately to make sure my permissions are correct, but when I try this, or `git+ssh` url, both fail with 

```
 ERROR  Command failed with exit code 128: git ls-remote git+ssh://git@github.com/mozilla-services/ads-sdk.git HEAD
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
So what I did instead was just copy the entire contents of `ads-sdk` repo into the `clients/web/node_modules` which may be the wrong thing to do and the cause of the error?

![Screenshot 2025-02-25 at 12 03 57 PM](https://github.com/user-attachments/assets/3cf40ae4-1c82-4d88-82d7-4117daec1245)

Reply:
Does this run locally?  If it does, perhaps we need to add PocketCI to the permissions on `@mozilla-services/ads-sdk`?

Reply:
This only works for local development, yes. We were hoping that using the git urls in `package.json` would work for a quick boostrap for local dev by leveraging our personal local github ssh keys to pull the sdk repo, but it didn't work for me for this attempt.

That's okay though, regardless we'll need to ultimately vend this to mozilla's NPM registry so that we can pull this dependency in a standard way, and a way that will work for CI and deploy builds. 

So if I understand your suggestion correctly, I think we don't want to worry about giving PocketCI permission to the sdk repo. I think we should just work on getting it vended properly. I'll kick off that process today.

Reply:
Welp, mozilla doesn't pay for the npm private packages, so that means when we push up to npm, we should consider that going public and prepare accordingly. I'm working with my manager Catherine and our Product Marketing lead Bryce to get that squared away ASAP since that blocks "real vending", which in turn blocks this PR from working anywhere but my local machine

Reply:
Hoping to be able to vend to npm on Wednesday 4/2 🤞